src/webportal/src/app/job-submission/job-submission.jsx

```diff
@@ -244,6 +244,9 @@ const JobSubmission = () => {
               {vcNames}
             );
             jobInfo.name = generateJobName(jobInfo.name);
+            if (get(jobConfig, 'extras.submitFrom')) {
+              delete jobConfig.extras.submitFrom;
```

Comment:
should not delete

Reply:
The config is cloned from a plugin. But if resubmit, the config should indicate it submitted from the default submission page. So I think remove is the correct behavior